src/styles/utils/_vars.scss

```diff
@@ -1 +1,8 @@
-$c-gray: #eee;
+$page-backgroun-color:#e4dfd3;
+$text-minor-color:#344f10;
+$menu-and-btn-background:#333;
+$left-title:#344f1080;
```

Comment:
Naming of the variables should be more descriptive. Variable name $left-title doesn't clearly indicate its purpose.